app.py

```diff
@@ -93,44 +116,79 @@ async def assets(path):
 AZURE_OPENAI_TOP_P = os.environ.get("AZURE_OPENAI_TOP_P", 1.0)
 AZURE_OPENAI_MAX_TOKENS = os.environ.get("AZURE_OPENAI_MAX_TOKENS", 1000)
 AZURE_OPENAI_STOP_SEQUENCE = os.environ.get("AZURE_OPENAI_STOP_SEQUENCE")
-AZURE_OPENAI_SYSTEM_MESSAGE = os.environ.get("AZURE_OPENAI_SYSTEM_MESSAGE", "You are an AI assistant that helps people find information.")
-AZURE_OPENAI_PREVIEW_API_VERSION = os.environ.get("AZURE_OPENAI_PREVIEW_API_VERSION", "2023-12-01-preview")
+AZURE_OPENAI_SYSTEM_MESSAGE = os.environ.get(
+    "AZURE_OPENAI_SYSTEM_MESSAGE",
+    "You are an AI assistant that helps people find information.",
+)
+AZURE_OPENAI_PREVIEW_API_VERSION = os.environ.get(
+    "AZURE_OPENAI_PREVIEW_API_VERSION", "2023-12-01-preview"
```

Comment:
You will probably want to pull the latest changes soon -- we recently updated to use the most recent preview API, so there are a few structural changes in the request / response contracts with the OYD service.